src/Microsoft.Health.Fhir.Liquid.Converter.UnitTests/Filters/DateFiltersTests.cs

```diff
@@ -26,9 +26,9 @@ public static IEnumerable<object[]> GetValidDataForAddHyphensDate()
         // We assume the local timezone is +08:00.
         public static IEnumerable<object[]> GetValidDataWithoutTimeZoneForAddHyphensDateWithUtcTimeZoneHandling()
         {
-            yield return new object[] { @"200101", "utc", @"2000-12" };
-            yield return new object[] { @"20010102", "utc", @"2001-01-01" };
-            yield return new object[] { @"19850101000000", "utc", @"1984-12-31" };
+            yield return new object[] { @"200101", "utc", new DateTime(2001, 1, 1) };
```

Comment:
Is it possible to add some test cases about half-hour time zone or summer time zone?

Reply:
Added half-hour time zone.